src/rf/apps/workers/process.py

```diff
@@ -146,6 +165,7 @@ def validate_image(self, record):
                         'timeout': time.time() + TIMEOUT_SECONDS,
                         'layer_id': layer_id
                     }
+                    log.info('Queue timeout job')
```

Comment:
There is a subtle difference, but it's about to get a little more confusing, since we just added a separate heartbeat job. I would like to consolidate these.